client/analysisui/iconsupport.css

```diff
@@ -0,0 +1,40 @@
+/*textbox sizing*/
+.jmv-option-icon[data-icon=ContinuousVariable] {
```

Comment:
i'm ok with dashes. we need some way to express groups/hierarchy, i could see us adding things in the future like:

    variable-nominaltext
    info-warning
    value-optional

thoughts?